daemon/algod/api/server/v2/handlers.go

```diff
@@ -945,6 +951,63 @@ func (v2 *Handlers) TealDryrun(ctx echo.Context) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
+// UnsetSyncRound removes the sync round restriction from the ledger.
+// (DELETE /v2/ledger/sync)
+func (v2 *Handlers) UnsetSyncRound(ctx echo.Context) error {
+	v2.Node.UnsetSyncRound()
+	return ctx.NoContent(http.StatusOK)
+}
+
+// SetSyncRound sets the sync round on the ledger.
+// (POST /v2/ledger/sync/{round})
+func (v2 *Handlers) SetSyncRound(ctx echo.Context, round uint64) error {
+	err := v2.Node.SetSyncRound(round)
+	if err != nil {
+		switch err {
+		case catchup.ErrSyncRoundInvalid:
+			return badRequest(ctx, err, errFailedSettingSyncRound, v2.Log)
+		default:
+			return internalError(ctx, err, errFailedSettingSyncRound, v2.Log)
+		}
+	}
+	return ctx.NoContent(http.StatusOK)
+}
+
+// GetSyncRound gets the sync round from the ledger.
+// (GET /v2/ledger/sync)
+func (v2 *Handlers) GetSyncRound(ctx echo.Context) error {
+	rnd := v2.Node.GetSyncRound()
+	if rnd == 0 {
+		return notFound(ctx, fmt.Errorf("sync round is not set"), errFailedRetrievingSyncRound, v2.Log)
+	}
+	return ctx.JSON(http.StatusOK, model.GetSyncRoundResponse{Round: rnd})
+}
+
+// GetLedgerStateDelta returns the deltas for a given round.
+// This should be a representation of the ledgercore.StateDelta object.
+// (GET /v2/deltas/{round})
```

Comment:
Is there any feature flagging? It looks like these features would be enabled right away.

Reply:
You're right--I noticed these were tagged incorrectly in the API spec causing them to be registered. I've reclassified them so that they won't be registered in the existing APIs.